### PR TITLE
[CWS] no error log for filtered rules and report rules in status

### PR DESCRIPTION
**Pull request by safchain**
### What does this PR do?

Avoid having error log for rules of another platform like error on linux for windows rules.

### Describe how to test/QA your changes

Shouldn't see any error in the log with the default policy

### Reviewer's Checklist

- [ ] If known, an appropriate milestone has been selected; otherwise the `Triage` milestone is set.
- [ ] Use the `major_change` label if your change either has a major impact on the code base, is impacting multiple teams or is changing important well-established internals of the Agent. This label will be use during QA to make sure each team pay extra attention to the changed behavior. For any customer facing change use a releasenote.
- [ ] A [release note](https://github.com/DataDog/datadog-agent/blob/main/docs/dev/contributing.md#reno) has been added or the `changelog/no-changelog` label has been applied.
- [ ] Changed code has automated tests for its functionality.
- [ ] Adequate QA/testing plan information is provided if the `qa/skip-qa` label is not applied.
- [ ] At least one `team/..` label has been applied, indicating the team(s) that should QA this change.
- [ ] If applicable, docs team has been notified or [an issue has been opened on the documentation repo](https://github.com/DataDog/documentation/issues/new).
- [ ] If applicable, the `need-change/operator` and `need-change/helm` labels have been applied.
- [ ] If applicable, the `k8s/<min-version>` label, indicating the lowest Kubernetes version compatible with this feature.
- [ ] If applicable, the [config template](https://github.com/DataDog/datadog-agent/blob/main/pkg/config/config_template.yaml) has been updated.

### CODE REVIEW

File: pkg/security/secl/rules/policy.go [hunk 1]
```diff
@@ -135,7 +136,10 @@ LOOP:
 			}
 		}
 
-		errs = multierror.Append(errs, &ErrRuleLoad{Definition: s.ruleDefinition, Err: s.err})
+		// do not report filtered rules
+		if !errors.Is(s.err, ErrRuleAgentFilter) {
+			errs = multierror.Append(errs, &ErrRuleLoad{Definition: s.ruleDefinition, Err: s.err})
```
[the remark this answers is not in the document]
Comment (safchain) [reply pkg/security/secl/rules/policy.go:141]: We still have the ruleset_loaded event. I'm not sure adding a log with a huge list of ignored rules will help us to understand why they have been ignored.  I would prefer to list as info the loaded rules, wdyt ?

[the remark this answers is not in the document]
Comment (safchain) [reply pkg/security/secl/rules/policy.go:141]: I'm going to add the rules status in the agent status report

**Review by pgimalac — APPROVED** [review]
LGTM for status

File: pkg/security/secl/rules/policy.go (hunk 1 above)
[the remark this answers is not in the document]
Comment (safchain) [reply pkg/security/secl/rules/policy.go:141]: @paulcacheux I added the status, could you have another look

**Review by lebauce — APPROVED** [review]

**Review by paulcacheux — APPROVED** [review]

**Pull request merged**